packages/cdk/lib/roles/policies/toil-batch-policy.ts

```diff
@@ -0,0 +1,33 @@
+import { PolicyDocument, PolicyStatement, Effect } from "aws-cdk-lib/aws-iam";
+
+export interface ToilBatchPolicyProps {
+  jobQueueArn: string;
+  // This is actually a pattern that matches all ARNs for potentially relevant
+  // definitions, since Toil makes its own definitions.
+  toilJobArn: string;
```

Comment:
toilJobDefinitionArnPattern would be a more meaningful name

Reply:
However, a similar change may want to be made to `cromwellJobArn`, which is also a pattern for engine-managed job definitions: https://github.com/aws/amazon-genomics-cli/blob/8bda67492b3cf143ffd5a0a8931bbcdf3e04992d/packages/cdk/lib/roles/cromwell-engine-role.ts#L17-L26. I don't think that would make sense to do in this PR.